src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -21,14 +21,35 @@ internal partial class BoundBadStatement
         protected override ImmutableArray<BoundNode> Children => this.ChildBoundNodes;
     }
 
+    partial class BoundFixedStatement
+    {
+        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Declarations, this.Body);
+    }
+
     partial class BoundLocalFunctionStatement
     {
         protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Body);
     }
 
-    partial class BoundFixedStatement
+    partial class BoundMethodGroup
```

Comment:
>BoundMethodGroup [](start = 18, length = 16)

This node is an expression, it belongs in Expressions.cs. Also, I believe [https://github.com/dotnet/roslyn/pull/21857](https://github.com/dotnet/roslyn/pull/21857) already includes proper implementation.